src/database/types.rs

```diff
@@ -0,0 +1,9 @@
+use crate::identity_tree::Hash;
+
+pub struct UnprocessedCommitment {
+    pub commitment:    Hash,
+    pub status:        String,
```

Comment:
IMO this should be a variant from Status
https://github.com/worldcoin/signup-sequencer/pull/514/files#diff-d2f33d7e543d3bd1fed57d73380ca787ce27cd63c5b40cf59f43ff0e3d29afc1R48